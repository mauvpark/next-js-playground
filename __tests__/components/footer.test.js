import renderer from "react-test-renderer";
import Footer from "../../components/footer";

it("<Footer /> should render only copyright text by Mauv", () => {
	const tree = renderer.create(<Footer />).toJSON();
	expect(tree).toMatchSnapshot();
});
