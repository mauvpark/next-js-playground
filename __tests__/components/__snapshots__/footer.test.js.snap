// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render only copyright text by Mauv 1`] = `
<footer>
  Copyrights all reserved by Mauv
</footer>
`;
